ftw/mail/indexers.py

```diff
@@ -0,0 +1,12 @@
+from ftw.mail.mail import IMail
+from ftw.mail.utils import get_date_header
+from plone.indexer.decorator import indexer
+from DateTime import DateTime
+
+
+@indexer(IMail)
+def Date(object_, **kw):
+    """indexes the userid of the object owner"""
+    email_date = get_date_header(object_.msg, "Date") or object_.created()
```

Comment:
Yes, I now return a DateTime directly from the get_date_header. The function is also used in the the mail view (mail_templates/view.pt), but it also works when passing a `DateTime` instead of a `str`
I also now use the `object_.get_date_header()` method for the indexing